test/inductor/test_foreach.py

```diff
@@ -37,14 +37,18 @@
     torch._foreach_sub,
     torch._foreach_div,
     torch._foreach_maximum,
+    torch._foreach_minimum,
+    torch._foreach_clamp_max,
+    torch._foreach_clamp_min,
     aten._foreach_copy,
 ]
 
 un_ops_under_test = [
     torch._foreach_reciprocal,
     torch._foreach_neg,
-    torch._foreach_abs,
     torch._foreach_sign,
+    torch._foreach_abs,
```

Comment:
Why move _foreach_abs? Was that intentional?

Reply:
Oops. Nope, it was just an artifact from having 2 branches. I can swap it back if strictly necessary. (Just don't want the CI to run again)

Reply:
It's ok, feel free to merge